### ISSUE TITLE
Move Issues from Spec Registries to DID Core

### ISSUE THREAD

**Issue by OR13** [w3c/did-core#367]
Not getting enough contribution to the following spec registries issues: https://github.com/w3c/did-spec-registries/issues?q=is%3Aissue+is%3Aopen+label%3Amove-to-did-core

proposing we move them to did core.

**Comment by msporny** [comment]
This is waiting on me to move things over.

**Comment by msporny** [comment]
I have still not done this.

**Comment by kdenhartog** [comment]
I've gone through them all and many of them look semi addressed at this point. I've left comments to get authors to respond if they still want to transfer the issues to did-core and if so I can do it @msporny (from the looks of it I can handle it with my permissions on the two repos)

**Comment by msporny** [comment]
@kdenhartog wrote:
>  I've left comments to get authors to respond if they still want to transfer the issues to did-core and if so I can do it @msporny (from the looks of it I can handle it with my permissions on the two repos)

Yes, please move them if you have permissions to do so @kdenhartog ... I am utterly failing to get to that item on my todo list. Thank you!

**Comment by brentzundel** [comment]
@kdenhartog can you update us on the status of this?

**Comment by kdenhartog** [comment]
The only issue that was left has been moved: https://github.com/w3c/did-core/issues/478

The rest have either been addressed in a PR and closed/pending close or were no longer relevant and closed. I suggest declaring this issue complete and will label it pending close.

**Comment by brentzundel** [comment]
No objections raised since marked pending close, closing.
